src/concurrent.cr

```diff
@@ -58,8 +58,11 @@ end
 #
 # 2.times { ch.receive }
 # ```
-def spawn(*, name : String? = nil, &block)
+def spawn(*, name : String? = nil, same_thread = false, &block)
```

Comment:
Just make sure nobody changes the contract from "stays on the same thread" to "is initially queued for the same thread" in case job stealing is implemented :D

Reply:
I just notice that `same_thread` should be typed as `Bool` here